src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/SchedulerConfiguration.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.config
+
+import net.javacrumbs.shedlock.core.LockProvider
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import org.springframework.scheduling.annotation.EnableScheduling
+import javax.sql.DataSource
+
+@Configuration
+@EnableScheduling
+@Profile("!test") // prevent scheduler running during integration tests
+@EnableSchedulerLock(
+  defaultLockAtLeastFor = "PT10M",
```

Comment:
should this me less is 10 mins too much

Reply:
yep 5 mins fine I think

Reply:
@AledWEvans  I have now updated the lock (atLeastFor and AtMostFor) to 5 mins.